src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset-group">
+        <legend>Personal information</legend>
+        <label class="input-group">Surname:
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-group" for="name">Name:
```

Comment:
Add an empty line between multiline siblings
Apply all over the page
```suggestion
        </label>
        
        <label class="input-group" for="name">Name:
```